dbt-cta/twilio/models/1_cta_base/alerts_base.sql

```diff
@@ -1,7 +1,11 @@
+{% set partitions_to_replace = [
```

Comment:
Yep - in the process of testing I came across a weird issue where the `alerts` stream (but seemingly no others?) kept delivering duplicate rows into the _airbyte_raw table, and it's a tiny table so easier to just full refresh overwrite instead of figuring out incrementality. 

Oh and now I'm reading your updated comment so maybe you already read my explanation lol, assuming I did explain this somewhere... but yes good flag, I'll reset that stream in prod and rerun to make sure it's all copacetic! (gonna wait for @jackwelty to look this over first tho)